lang/cs/Org.Apache.REEF.Common/Files/REEFFileNames.cs

```diff
@@ -229,6 +230,11 @@ public string GetBridgeExeConfigPath()
             return Path.Combine(REEF_BASE_FOLDER, BRIDGE_EXE_CONFIG_NAME);
         }
 
+        /// <summary>
+        /// </summary>
+        /// <returns>File name that contains the dfs path for the DriverHttpEndpoint</returns>
+        public string DriverHttpEndpoint { get { return DRIVER_HTTP_ENDPOINT_FILE_NAME; } }
```

Comment:
This needs to have an `[Unstable]` annotation, as it is likely to go away once we solve this for good.